server/src-lib/Hasura/GraphQL/Schema/Common.hs

```diff
@@ -0,0 +1,54 @@
+module Hasura.GraphQL.Schema.Common
```

Comment:
`Common` for lack of a better name. These functions/values are imported by more than one sub-module under `Schema`.